src/doc/rustc/src/platform-support/wasm32-unknown-emscripten.md

```diff
@@ -0,0 +1,98 @@
+# `wasm32-unknown-emscipten`
+
+**Tier: 2**
+
+The `wasm32-unknown-emscripten` target is a WebAssembly compilation target which
+uses the Emscripten compiler toolchain. Unlike the
+[`wasm32-unknown-unknown`](./wasm32-unknown-unknown.md) target, which only has
+partial support for the Rust standard library, `wasm32-unknown-emscripten`
+integrates with JavaScipt on the web to provide a more featureful standard library.
+This target is often used when compiling for the web, accessing web APIs, or
+interoperating with C and C++ code.
+
+One existing user of this target (please feel free to edit and expand this list
+too) is the [`pyodide` project](https://github.com/pyodide/pyodide) which provides
+a Python runtime in WebAssembly using Emscripten and compiles Python extension
+modules written in Rust to the `wasm32-unknown-emscripten` target.
+
+## Target maintainers
+
+When this target was added to the compiler, platform-specific documentation here
+was not maintained at that time. This means that the list below is not
+exhaustive, and there are more interested parties in this target. That being
+said, those interested in maintaining this target are:
+
+- TODO
+
+## Requirements
+
+This target is cross-compiled. The Emscripten compiler toolchain must be
+installed to link Rust-compiled code with Emscripten's libc and other C/C++
+code.
```

Comment:
Something here should probably link to https://emscripten.org/, https://emscripten.org/docs/getting_started/downloads.html, or the relevant github repos.

Reply:
Done